community-operators/federation/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,190 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation.v0.0.5
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Integration & Delivery"
+    description: "Kubernetes Federation namespace-scoped installation"
+spec:
+  displayName: Federation
+  description: |
+    This deploys Federation in a namespace-scoped configuration. After
+    installation, you must enable the API types you want Federation to manage
+    using `kubefed2 enable`.
+
+    Note, you must enable federation support for namespaces in this
+    configuration before other types will work. Use `kubefed2 enable namespaces`.
+  version: 0.0.5
+  maturity: alpha
+  provider:
+    name: Red Hat
+  labels:
+    alm-owner-federation: federation
+    alm-status-descriptors: federation.v0.0.5
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: false
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: federation-controller-manager
+        rules:
+        - apiGroups:
+          - clusterregistry.k8s.io
+          resources:
+          - clusters
+          verbs:
+          - "*"
+        - apiGroups:
+          - core.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - primitives.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - multiclusterdns.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - scheduling.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - "*"
+      # TODO(font): use statefulset
+      deployments:
+      - name: federation-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: federation-controller-manager
+          template:
+            metadata:
+              labels:
+                app: federation-controller-manager
+            spec:
+              containers:
+                - name: controller-manager
+                  image: quay.io/pmorie/origin-federation-controller:v4.0.0
```

Comment:
Could we use an image is not from the personal registry?

Reply:
Whoops! excellent catch :)

Reply:
/hold